charts/zalenium/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: zalenium
 version: 0.0.1
-appVersion: 3.4.0
+appVersion: 3.141.59
```

Comment:
should be aligned with version in index.yaml

Reply:
Done.